blueprint/abac/platform.md

```diff
@@ -386,86 +396,105 @@ Microsoft StaffHub | Off
+`Azure Active Directory | Groups | All groups > rol-Agency-Administrators | Licenses > Microsoft 365 E5`
 
 Item | Configuration
 --- | ---
 Azure Active Directory Premium P1 | On
 Azure Active Directory Premium P2 | On
-Azure Advanced Threat Protection | Off
-Azure Information Protection Premium P1 | Off
-Azure Information Protection Premium P2 | Off
-Azure Rights Management | Off
-Customer Lockbox | Off
+Azure Advanced Threat Protection | On
+Azure Information Protection Premium P1 | On
+Azure Information Protection Premium P2 | On
+Azure Rights Management | On
+Common Data Service | Off
+Common Data Service for Teams | Off
+Customer Lockbox | On
+Data Classification in Microsoft 365 | On
 Exchange Online (Plan 2) | Off
-Flow for Office 365 | Off
-Information Barriers | Off
-Information Protection for Office 365 - Premium | Off
-Information Protection for Office 365 - Standard | Off
+Graph Connectors Search with Index | Off
+Information Barriers | On
+Information Protection for Office 365 - Premium | On
+Information Protection for Office 365 - Standard | On
 Insights by MyAnalytics | Off
-Microsoft 365 Advanced Auditing | Off
-Microsoft 365 Apps for enterprise | Off
-Microsoft 365 Audio Conferencing | Off
+M365 Communication Compliance | On
+Microsoft 365 Advanced Auditing | On
+Microsoft 365 Apps for enterprise | On
+Microsoft 365 Audio Conferencing | On
+Microsoft 365 Defender | On
 Microsoft 365 Phone System | Off
-Microsoft Azure Multi-Factor Authentication | Off
+Microsoft Azure Multi-Factor Authentication | On
+Microsoft Bookings | Off
 Microsoft Cloud App Security | On
-Microsoft Communications Compliance | Off
-Microsoft Communications DLP | Off
+Microsoft Communications DLP | On
 Microsoft Customer Key | Off
-Microsoft Data Investigations | Off
-Microsoft Defender Advanced Threat Protection | Off
+Microsoft Data Investigations | On
+Microsoft Defender For Endpoint | On
+Microsoft Defender for Identity | On
+Microsoft Defender for Office 365 (Plan 2) | On
+Microsoft Endpoint DLP | On
+Microsoft Excel Advanced Analytics | Off
 Microsoft Forms (Plan E5) | Off
-Microsoft Information Governance | Off
-Microsoft Insider Risk Management | Off
+Microsoft Information Governance | On
+Microsoft Insider Risk Management | On
 Microsoft Intune | Off
 Microsoft Kaizala Pro | Off
-Microsoft ML-based classification | Off
+Microsoft ML-based classification | On
 Microsoft MyAnalytics (Full) | Off
 Microsoft Planner | Off
-Microsoft Records Management | Off
+Microsoft Records Management | On
 Microsoft StaffHub | Off
 Microsoft Stream for O365 E5 SKU | Off
 Microsoft Teams | Off
-Office 365 Advanced eDiscovery | Off
-Office 365 Advanced Threat Protection (Plan 2) | Off
-Office 365 Cloud App Security | Off
-Office 365 Privileged Access Management | Off
-Office 365 SafeDocs | Off
+Office 365 Advanced eDiscovery | On
+Office 365 Cloud App Security | On
+Office 365 Privileged Access Management | On
+Office 365 SafeDocs | On
```

Comment:
Just wondering why some of these features have been changed to enabled. Are they dependencies from the products we already cover?

Reply:
I have updated the list to remove the Microsoft 365 Apps which was added by mistake. 

The additional enablement of services relate ensuring that administrator accounts are covered by Microsoft protection services such as defender for identity. Administrators would also likely be gaining a benefit from these services relating to advanced detection methods. As Microsoft increase the service targeting - users not licensed for security products may be exposed to risk. I have attached some details around MS licensing for security products - https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance